background/contentScript.js

```diff
@@ -10,3 +10,9 @@ window.addEventListener('message', function(event) {
     });
   }
 }, false);
+
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  if (request.from && request.from === 'popup') {
+    window.postMessage({from: 'content', data: request}, "*");
+  }
+});
```

Comment:
Extension cannot directly send data from the popup screen to the website.
Therefore, it should be sent to popup -> contentscript.js -> website.